### PR TITLE
[sflow + dropmon] added missing patch for the sFlow + dropmon feature.

### PR DESCRIPTION
**Pull request by vadymhlushko-mlnx**
Added a patch with changes required for the `sFlow + dropmon` feature.

`#define PSAMPLE_MD_EXTENDED_ATTR   1` should be declared in the `sonic-linux-kernel` repo but used in [Switch-SDK-drivers/drivers/net/sx_netdev/sx_psample.c](https://github.com/Mellanox/Switch-SDK-drivers/blob/sx_sdk_4_5_1200_4.5.1208/drivers/net/sx_netdev/sx_psample.c)

Signed-off-by: Vadym Hlushko <vadymh@nvidia.com>

### CODE REVIEW

**Comment by vadymhlushko-mlnx** [comment]
@saiarcot895 could you please review?

**Comment by vadymhlushko-mlnx** [comment]
> I see that the buildimage PR depends on this. Since the other PR is already merged, does it break buildimage?

Yes, it brakes the buildimage

File: patch/0003-psample-added-define-PSAMPLE_MD_EXTENDED_ATTR.patch [hunk 1]
```diff
@@ -0,0 +1,25 @@
+From 7dbf2689eb841c51dca4dad51b0941c06aa09e26 Mon Sep 17 00:00:00 2001
+From: Vadym Hlushko <vadymh@nvidia.com>
+Date: Mon, 11 Apr 2022 15:41:46 +0000
+Subject: [PATCH 3/3] psample: added define PSAMPLE_MD_EXTENDED_ATTR, used in SDK driver sx_psample.c
```
[the remark this answers is not in the document]
Comment (vadymhlushko-mlnx) [reply patch/0003-psample-added-define-PSAMPLE_MD_EXTENDED_ATTR.patch]: Done

File: patch/0003-psample-added-define-PSAMPLE_MD_EXTENDED_ATTR.patch [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
```
[the remark this answers is not in the document]
Comment (vadymhlushko-mlnx) [reply patch/0003-psample-added-define-PSAMPLE_MD_EXTENDED_ATTR.patch:8]: Done

**Review by saiarcot895 — APPROVED** [review]

File: patch/0003-psample-added-define-PSAMPLE_MD_EXTENDED_ATTR.patch (hunk 2 above)
[the remark this answers is not in the document]
Comment (paulmenzel) [reply patch/0003-psample-added-define-PSAMPLE_MD_EXTENDED_ATTR.patch:8]: Maybe: Define the macro `PSAMPLE_MD_EXTENDED_ATTR` …

But a very minor nit.

**Review by paulmenzel — COMMENTED** [review]
Please also update the merge/pull request summary/title.

**Comment by vadymhlushko-mlnx** [comment]
> Please also update the merge/pull request summary/title.

Done

**Pull request merged**